charts/edge-webrtc-service/values.yaml.Example

```diff
@@ -0,0 +1,65 @@
+namespace: "hanwha"
+# fullnameOverride: imagename
+
+image:
+  replicaCount: 1
+  # repository: ghcr.io/htaic/video-edge-webrtc-service
+  repository: ghcr.io/htaic/video-edge-webrtc-service
+  pullPolicy: Always
+  containerPort: 3000
+  tag: "latest"
+  domainPath: ""
+  livenessProbe: "api/v1/livez"
+  readinessProbe: "api/v1/readyz"
```

Comment:
you have hard coded in the deployment liveness/readiness so you can remove these